app/src/main/res/values/strings.xml

```diff
@@ -349,7 +351,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="example_of_a_markdown_table">Example of a Markdown table</string>
     <string name="delete_lines">Delete lines</string>
     <string name="unordered_list">Unordered list</string>
-    <string name="ordered_list">Ordered list</string>
```

Comment:
This is used for description at UI Ordererd list button too as far I know. Please have a new renumber_ordered_list key

Reply:
Long press of the ordered list action triggers a renumber ordered list for the current list. This is quite useful. Changing the above string changes the long-press message.

Reply:
@harshad1 still please use new res renumber_ordered_list if you change the string, OR revert that line of change.

Reason is that translation services/Crowdin act bad on string change, and if the rate is matching may reuse the old one in a given language automatically